general/OKRs/README.md

```diff
@@ -0,0 +1,156 @@
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Publish blogposts at a regular pace
+  * Plan weekly schedule for blogs
+  * Publish 8 blog posts during Q1
+* Start cross posting to Medium
+  * Publish 10 posts to Medium
+  * Once we have enough analytics decide whether it's worth cross posting or simply migrating to medium. 
+* Start publishing videos
+  * Create YouTube channel for source{d}
+  * Create monthly schedule for videos and themes
+  * Write 4 detailed scripts
+  * Publish 2 videos
+* Publish a page where we show all source{d} related videos
+  * Redesign talks page
+  * Include all old videos and the ones we'll publish in the meanwhile
+  * Include 5 talks by non source{d} employees
+* Redefine how we use social media and community Slack
+  * Gather statistics of current usage and views on social media
+  * Write a mission statement for our social media and formalize who has access/responsibilities
+  * Once we have statistics on community size, establish new OKRs on growth
+
+### Academia:
+
+* Getting academia to use our tech
+  * source{d} technologies mentioned in at least one research paper by non-source{d} researcher
+* Understand usecases/workflow
+  * Identify 3 common needs for the researchers above
+  * Prioritize those needs and tackle at least one.
+* Gather inputs on what 'ImageNet for Code' is to stakeholders in the community
+  * Write a blog post on what 'ImageNet for Code' in collaboration with a non source{d} researcher
+  * Make a video about this
+  * Publish it and gather feedback from other researchers
+  * Key Result 3
+* Deepen our relationship with universities
+  * Identify and partner with 4 researchers
+  * Meet monthly with each of them and analyze their needs
+  * Keep notes and minutes for each one of these meetings
+
+### Research
+
+* Write scientific papers
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Research structural embeddings
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Explore new directions and emerging papers
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Engineering:
+
+* Collect, store and process 10s of millions of repositories using the pipeline in Paris
+  * Improve the borges performance to be able to download all the repositories in 1 month with 60 machines.
+  * Scale our infrastructure to support this - a cluster of 60 machines.
+  * Develop a system to keep this updated with a delay of 1 week.
+* Launch a production level engine
+  * Support large scale deployments - 10s of millions of repositories
+  * Support performance large scale deployments (using indexes or not) to be able to execute the 90% of queries under 30mins of execution.
+  * Improve the error reporting to minimize the number of issues open.
+* Significantly improve the engineering team workflow
+  * Deliver products on time, without commit other teams.
+  * Improve the inter-team workflows.
+  * Improve the communication at all levels.
```

Comment:
How could we mesure this two "improve" items.

Reply:
@mcuadros tagging you here. 

How to measure this is *hard*!